source/extensions/filters/network/common/redis/client.h

```diff
@@ -134,6 +140,18 @@ class Config {
    * @return timeout for batching commands for a single upstream host.
    */
   virtual std::chrono::milliseconds bufferFlushTimeoutInMs() const PURE;
+
+  /**
+   * @return interval for checking the state of draining upstream connections.
+   *
+   */
+  virtual std::chrono::milliseconds upstreamDrainPollIntervalInMs() const PURE;
+
+  /**
+   * @return the maximum number of upstream connections to unknown hosts when enableRedirection() is
+   * true.
+   */
+  virtual uint32_t maxUpstreamUnknownConnections() const PURE;
```

Comment:
Can you add a note that this is effectively the max scale up size?  For example if we scale a cluster from 128 to 256 and maxUpstreamUnknownConnections is 100, then requests might start to fail.

Reply:
no problem